windows/vNext/ReactNativeVideoCPP/ReactVideoView.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
are you intending this file to be licensed separately from the rest of the project?

Reply:
Removed  the copy right text, thanks for the catch.